src/BaseTable.tsx

```diff
@@ -1082,7 +1180,7 @@ export interface IBaseTableProps<T = any> {
   /**
    * Custom scrollbar size measurement
    */
-  getScrollbarSize?: Function;
+  getScrollbarSize?: () => number;
```

Comment:
I think this gets a boolean, not sure thoigh, but the function defined in utils with the same name gets a boolean

Reply:
But it is expected as a number in:
> BaseTable.tsx, `line:628`

```typescript
const scrollbarSize = this.props.getScrollbarSize();
if (scrollbarSize > 0) {
       ...
}
```